src/uu/stty/src/stty.rs

```diff
@@ -588,6 +653,22 @@ fn apply_char_mapping(termios: &mut Termios, mapping: &(SpecialCharacterIndices,
     termios.control_chars[mapping.0 as usize] = mapping.1;
 }
 
+fn apply_special_setting(setting: &SpecialSetting, fd: i32) -> nix::Result<()> {
+    let mut size = TermSize::default();
+    unsafe { tiocgwinsz(fd, &raw mut size)? };
+    match setting {
+        SpecialSetting::Rows(n) => {
+            size.rows = *n;
+            unsafe { tiocswinsz(fd, &raw mut size)? };
+        }
+        SpecialSetting::Cols(n) => {
+            size.columns = *n;
+            unsafe { tiocswinsz(fd, &raw mut size)? };
+        }
+    }
```

Comment:
A detail: the `unsafe` block is identical in both cases and so I would move it outside of the `match`.